### DISCUSSION TITLE
Alternative name for `Project`

### DISCUSSION THREAD

**Discussion by jens-br** [nest-desktop/nest-desktop#475] [Ideas]
While adding some more documentation, I found that the current naming might be a bit misleading:
The word "project" can mean a project of any type. Since we can use create, edit and run functionalities also in the models section, this section also offers a use case, which could be described with "project" as well. This problem might increase when the Model section is extended in the future.
Therefore, it might be a bit helpful to have another title for the current "Project" section. At first I thought of "Simulation", but maybe the width of the sidebar would be a bit small for that?

**Comment by babsey** [comment]
An appropriate suggestion to rename `Project` would be `Experiment`.

**Comment by babsey** [comment]
Closed because it is duplicate of #403.
